src/renderer/wddmcon/WddmConRenderer.cpp

```diff
@@ -248,10 +248,10 @@ bool WddmConEngine::IsInitialized()
             NewChar = &_displayState[rowIndex]->New[colIndex];
 
             OldChar->Character = NewChar->Character;
-            OldChar->Atribute = NewChar->Atribute;
+            OldChar->Attribute = NewChar->Attribute;

```

Comment:
Nnnn. This one goes upstream. Can you revert and file a separate issue for any `Atribute` to `Attribute` that is on `WddmCon*` or `Bgfx*` related files (that is, any of them that are off of a `CD_IO_CHARACTER` struct as I have to go fix that in the driver declaration inside Windows too.)

Reply:
This is now #4469

@miniksa: Are all renderers in this same boat? In trying to rebase I ended up w/ additional changes to files as part of 5cde86c

Reply:
No, just those two (`WddmConEngine` and `BgfxEngine`) because they're special to bring-up editions of Windows that we don't ship. The other renderers don't run data through the driver in the `CD_IO_CHARACTER` interface.